.github/workflows/build-test.yml

```diff
@@ -245,10 +245,11 @@ jobs:
         uses: actions/upload-artifact@v4
         if: always()
         with:
-          name: e2e-artifacts-pull-request-${{ github.event.pull_request.number }}-commit-${{ github.sha }}
+          name: e2e-artifacts-pull-request-${{ github.event.pull_request.number }}-commit-${{ github.sha }}-${{ matrix.tests.name }}
           path: |
             ./packages/arb-token-bridge-ui/cypress/videos
             ./packages/arb-token-bridge-ui/cypress/screenshots
+          if-no-files-found: 'ignore'
```

Comment:
https://github.com/actions/upload-artifact?tab=readme-ov-file#inputs

this is to remove these warnings:
<img width="1313" alt="image" src="https://github.com/user-attachments/assets/a21bbdda-5f11-4914-b978-30f5e1f414ee">